yarn.lock

```diff
@@ -2154,13 +2154,6 @@
   dependencies:
     "@types/node" "*"
```

Comment:
Should that not be a dev dependency?

Reply:
yarn.lock doesn't discriminate between dev dependencies and non dev dependencies. Package.json is correct i think